mappings/net/minecraft/world/biome/DefaultBiomes.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_1972 net/minecraft/world/biome/DefaultBiomes
```

Comment:
More like `BuiltinBiomes`?

Reply:
I named it after `DefaultBiomeCreator` but I suppose they aren't intrinsically related, so I'll change it.